packages/sdk/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [1.0.0-rc.1`] - 2024-11-01
+
+### Changed
+
+- Renamed the server-side client class from `ServerClient` to `BackendClient` to
```

Comment:
Do you use a Markdown linter locally? I use one in VSCode but it doesn't break after 80 (?) characters like yours does, so when I edit these files, it always overwrites your changes. Happy to keep this formatting for dev readability but just want to make sure it's enforced for everyone.

Reply:
Yeah, I use **markdownlint** in VSCode, which is based off of another engine. I just added a Github workflow to lint the markdown files in this directory, and see if it's useful.